Ecommerce/index.css

```diff
@@ -1 +1,94 @@
-/* Write code below for ECOMMERCE TASK */
+/* Write code below for ECOMMERCE TASK */
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Welcome to My Shop</title>
+    <style>
+        body {
+            font-family: Arial, sans-serif;
+            margin: 0;
+            padding: 0;
+            background-color: #f5f5f5;
+            color: #333;
+        }
+        header {
+            background-color: #333;
+            color: #fff;
+            padding: 10px 20px;
+            text-align: center;
+            position: relative; /* Added position relative */
+        }
+        .brand-image {

```

Comment:
Positioning the brand image absolutely within the header is a good technique. Consider grouping this position-related CSS properties together for better code organization.